applications/launchpad/gui-react/src/containers/WalletPasswordWizard/types.ts

```diff
@@ -0,0 +1,4 @@
+export interface WalletPasswordWizardProps {
+  submitBtnText?: string
+  onSuccess?: () => void
```

Comment:
I can understand why the above is optional, you could have a default 'submit' text, but onSuccess looks like a required prop to me

Reply:
Not necessary in the Wizard, because on form submission, the container dispatches the action and changes the Redux state. So the parent component not necessary need to know whether form submit was successful, it can just observe the Redux state with some selector.